src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -133,6 +135,15 @@ internal class VisualizerInfo
             public VisualizerType Visualizer { get; private set; }
             public Dictionary<string, string> ScopedNames { get; private set; }
 
+            public IEnumerable<(string,int)> GetUIVisualizers()
```

Comment:
>(string,int) [](http://example.com/codeflow?start=31&length=12)

nit: we use this in a bunch of spots. Probably better to create a struct for it.